### ISSUE TITLE
Feature Request and Bug

### ISSUE THREAD

**Issue by Github-com-foss** [SimpleMobileTools/Simple-Music-Player#297]
1. Add Swipe Down Gesture to back instead of back button.
2. Add View and Edit Audio Tagger.
3. Pls add custom color value option in customize theme.
4. Add third color (accent bar ) option in customize theme.
5. Motion (movable) if artist name is long.
6. (Notice Bug) when enter app 5 to 6 sec blank screen to load...Most of music player loading the music at first only after install the app. Not every every time to load the music so pls fix it.

Some pictures are shuttle+ player for suggestions
(https://github.com/timusus/Shuttle)

Reference picture added Below

![1](https://user-images.githubusercontent.com/75672019/102224621-34a02500-3f0c-11eb-829b-e095cb180c73.jpg)
![2](https://user-images.githubusercontent.com/75672019/102224627-37027f00-3f0c-11eb-941d-1ef443c493bb.jpg)
![3](https://user-images.githubusercontent.com/75672019/102224631-3833ac00-3f0c-11eb-8524-9ff01489d1b4.jpg)
![4](https://user-images.githubusercontent.com/75672019/102224633-38cc4280-3f0c-11eb-9c93-ca851aaa9a63.jpg)
![4(4)](https://user-images.githubusercontent.com/75672019/102224637-39fd6f80-3f0c-11eb-8c3b-17fc3b931f22.jpg)
![5](https://user-images.githubusercontent.com/75672019/102224641-3b2e9c80-3f0c-11eb-8499-4f5e32e1f586.jpg)
![6](https://user-images.githubusercontent.com/75672019/102224646-3c5fc980-3f0c-11eb-8356-f2df9f573b1a.jpg)

**Comment by tibbi** [comment]
create a separate issue per request if it hasnt been reported yet please, it cannot be managed like this
